.github/workflows/docker-dev-release.yml

```diff
@@ -44,6 +45,14 @@ jobs:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Login to Gcp Artifact Registry
+        uses: docker/login-action@v3
+        with:
+          registry: us-central1-docker.pkg.dev
+          username: _json_key
```

Comment:
does it have to be `_json_key` ? Can we have a user "github_actions" for example?

Reply:
https://github.com/docker/login-action?tab=readme-ov-file#google-artifact-registry-gar has two ways to authenticate to artifact registry. `json_key` is one of them. Not sure what you meant by user "github_actions".  Could you explain?